### ISSUE TITLE
icinga -manage - Invoke-IcingaCommand catches an error: "Data at the root level is invalid."

### ISSUE THREAD

**Issue by stevie-sy** [Icinga/icinga-powershell-framework#273]
While testing the experimental feature "icinga -mange" with PR #270 and the current master the framework catches following error:
![image](https://user-images.githubusercontent.com/38723488/120438423-a4528100-c381-11eb-871b-6570a33f4e78.png)

**Comment by LordHepipud** [comment]
Thank you for the report. I never saw a message like this before. Could you please share your environment data, like PowerShell version any everything related?

* PowerShell Version used (`$PSVersionTable.PSVersion`):

* Operating System and version (`Get-IcingaWindowsInformation Win32_OperatingSystem | Select-Object Version, BuildNumber, Caption`):

**Comment by stevie-sy** [comment]
Sorry for not writting the environment data. Here are the required information:

- W2012R2
```
Version                                                         BuildNumber                                                     Caption                                                       
-------                                                         -----------                                                     -------                                                       
6.3.9600                                                        9600                                                            Microsoft Windows Server 2012 R2 Standar
```

```
Major  Minor  Build  Revision
-----  -----  -----  --------
4      0      -1     -1 
```

- W2016:
```
Version    BuildNumber Caption                               
-------    ----------- -------                               
10.0.14393 14393       Microsoft Windows Server 2016 Standard
```
```
Major  Minor  Build  Revision
-----  -----  -----  --------
5      1      14393  4402
```

- W2019:
```
Version    BuildNumber Caption                               
-------    ----------- -------                               
10.0.17763 17763       Microsoft Windows Server 2019 Standard
```

```
Major  Minor  Build  Revision
-----  -----  -----  --------
5      1      17763  1852  
```


This message doesn't come with with the command _icinga_, only with using the parameter _-manage_. BUT without the parameter it looks like the command is in an endless loop. Because you can't write anything to the icinga-console.

Also interesting with W2012R2 you'll see a lot more error message during the loop:
![image](https://user-images.githubusercontent.com/38723488/120980183-a8faa900-c776-11eb-9d14-20dc5a4ca233.png)


Anyway, with every (windows) version you have to use the stop button in the ISE. 

![image](https://user-images.githubusercontent.com/38723488/120980439-e828fa00-c776-11eb-88d0-4a8f76d290d9.png)

With Plugin/Framework-Version 1.4 the error messages come also with the command _icinga_:
![image](https://user-images.githubusercontent.com/38723488/120980823-46ee7380-c777-11eb-97af-1aafc7e11dd0.png)


I thought also that I forgot to run the commands 
```
Get-ChildItem -Path 'C:\Program Files\WindowsPowershell\Modules\icinga-powershell-framework' -Recurse | Unblock-File
Get-ChildItem -Path 'C:\Program Files\WindowsPowershell\Modules\icinga-powershell-plugins' -Recurse | Unblock-File
```
during the update (it's easy to forget something on a testing server, while testing things :-) ). But it's the same ... 

If you need more, tell me ;-)

**Comment by LordHepipud** [comment]
The error occurs on all 3 machines? The very last screenshot with the .psd1 was fixed with 1.5.0m but I wonder why 1.5.0 throws different errors.

You are starting the PowerShell in Non-Interactive mode? After some research, this is a start argument to start a PowerShell with.

If you open the PowerShell, you can run this command to check if it is set to true or not.

```powershell
[Environment]::GetCommandLineArgs().Contains('-NonInteractive')
```

If the PowerShell is set to start as non-interactive, all read prompts are not available, which might cause the issue with the management console, as it requires to read user input from the CLI.

**Comment by stevie-sy** [comment]
That's it. I tested this on every three test servers (W2012R2 - W2019). It's everywhere the same result. Only with starting PowerShell with the parameter "-NonInteractive" it works like expected:
![image](https://user-images.githubusercontent.com/38723488/121377680-d5fbb700-c942-11eb-9ae6-ff6b933455ee.png)

It look likes you have to check if the return value is "true". If not write a hint and exit. 

The last screenshot was just to compare with the version 1.4.x that we have a similar behavior here (just ignore the issue with the psd-file which you have still fixed in 1.5.0 ) ;-)
I tested this also with 1.4.x and the parameter "-NonInteractive". Also here it works as expected.

With this knowledge I can test the command "icinga -manage" for you :-)
